sys/log/full/src/log_fcb.c

```diff
@@ -274,19 +274,19 @@ log_fcb_start_append(struct log *log, int len, struct fcb_entry *loc)
 }
 
 /**
- * Calculates the number of message body bytes that should be included after
- * the entry header in the first write.  Inclusion of body bytes is necessary
- * to satisfy the flash hardware's write alignment restrictions.
+ * Calculates the number of bytes that should be included after
+ * the entry header or trailer in the first write. Inclusion of bytes
+ * is necessary to satisfy the flash hardware's write alignment restrictions.
```

Comment:
Comment is not strictly correct `first write` only applies to bytes after header. Trailer is written in second write.

Reply:
Fixing it.